### PR TITLE
sign up page added

### PR DESCRIPTION
**Pull request by sukrugultop**

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,24 +1,12 @@
...
-                    className="App-link"
-                    href="https://reactjs.org"
-                    target="_blank"
-                    rel="noopener noreferrer">
-                    <Button variant="contained" color="primary">
-                        Learn React
-                    </Button>
-                    
-                </a>
+                <SignupPage></SignupPage>
```
Comment (ergunsh) [inline frontend/src/App.js:9]: We can use self closing tags:
```suggestion
                <SignupPage/>
```

**Comment by ergunsh** [comment]
Can you rebase `develop` into this branch to run `now` checks; which will deploy frontend of the app and we will be able to see your changes visually. That way, reviewing the PR will be easier. 😅

If you don't mind, I can also do that 😂

**Comment by sukrugultop** [comment]
If it is urgent, I would be very pleased with your doing, because I cannot do that in 6-7 hours.

**Review by huriozdmr — APPROVED** [review]

**Review by ergunsh — COMMENTED** [review]
Trying new github suggestion thing, it rocks; give it a try:

**Comment by ergunsh** [comment]
I think, we can close this in favor of: https://github.com/bounswe/bounswe2018group2/pull/139; wdyt @sukrugultop ?

**Pull request closed without merging**